tests/fixtures.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+from bitshares import BitShares
+from bitshares.instance import set_shared_blockchain_instance
+from bitshares.account import Account
+import os
+nodes = [
+    "ws://api.weaccount.cn:9777",
+    "ws://api.btspp.io:9777",
+    "wss://testnet.bts.dcn.cx/ws",
+    "wss://testnet.dex.trading/",
+    "wss://testnet-eu.bitshares.apasia.tech/ws",
+    "wss://testnet.bitshares.apasia.tech/ws",
+    "wss://node.testnet.bitshares.eu"
+]
+
+
+def init_default(account_name='', wif=''):
+    # default wifs key for testing
+    wifs = [wif]
+    # bitshares instance
+    bitshares = BitShares(
+        nodes, keys=wifs, nobroadcast=False, num_retries=1
+    )
+
+    # Set defaults
+    set_shared_blockchain_instance(bitshares)
+    bitshares.set_default_account(account_name)
+
+    # Ensure we are  going to transaction anythin on chain!
+    if bitshares.nobroadcast:
+        raise AssertionError('No broadcasting, no actual transactions!')
+
+
+def fixture_data_RO():
```

Comment:
Why you don't using pytest fixtures? https://docs.pytest.org/en/latest/fixture.html

Reply:
Keep it as simple as possible, I think. It's very professional of you to write like this, but what if someone else wants to run your test program?

Reply:
Sorry but pytest fixtures are cool to do nice elegant tests, I don't think we need to invent bicycles.

Reply:
I can use your code as a framework. But I have a puzzle: how do other people run your code?

Reply:
Ideally your tests should be based on my work, the problem is that it's not merged yet, and it requires new methods of pybitshares which wasn't merged too yet. So you'll have troubles running it.

When it will be finished, it could be run as regular, by `pytest`.